doc/rose-rug-suites-I.html

```diff
@@ -633,51 +633,57 @@ <h3 class="alwayshidden">Jinja2 rose-suite.conf Metadata Example</h3>
     </div>
 
     <div class="slide">
+      <h2 id="utc-mode">UTC mode</h2>
+
+      <p>We recommend running all suites in UTC mode. E.g.:</p>
+      <pre class="prettyprint lang-cylc">
+[cylc]
+    UTC mode = True # Ignore DST
+</pre>
+    </div>
+
+    <div class="slide">
       <h2 id="event-hooks">Event Hooks</h2>
 
-      <p>Suites also need some standard Rose <em>event hooks</em> to construct
-      the suite log viewer (more later) and report or shutdown on failure.</p>
+      <p>Suites can have event handlers to report events or shutdown on
+      failure.</p>
 
-      <p><strong>The following should be in every Rose suite:</strong></p>
       <pre class="prettyprint lang-cylc">
 [cylc]
     UTC mode = True # Ignore DST
+    # abort if any task fails = True
     [[event hooks]]
```

Comment:
This is still `event hooks`, at the suite level. We have intention to make this consistent with that the task level settings. cylc/cylc#1528